distributed/cli/utils.py

```diff
@@ -69,3 +71,12 @@ def uri_from_host_port(host_arg, port_arg, default_port):
     addr = unparse_address(scheme, loc)
 
     return addr
+
+
+def create_ssl_context(certfile, keyfile):
+    if certfile and keyfile:
+        ssl_ctx = ssl.create_default_context(ssl.Purpose.SERVER_AUTH)
```

Comment:
The theoretical difficulty here is that both client and server connections will be created using the same context, even though you specified `SERVER_AUTH`. Nowadays this doesn't matter so much, but it may matter in subsequent revisions of the standard `ssl` module.